Three/include/CGAL/Three/Viewer_interface.h

```diff
@@ -136,6 +136,7 @@ class VIEWER_EXPORT Viewer_interface : public QGLViewer, public QOpenGLFunctions
   void requestContextMenu(QPoint global_pos);
   //!Defined automatically in moc.
   void selectedPoint(double, double, double);
+  void pointSelected(QPoint p);
   //!Defined automatically in moc.
```

Comment:
Right, both are points, a 2D point and a 3D point. But with those names, I am curious to see how you will document the two functions. A name with "pixel" would lift the ambiguity.